solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -845,6 +852,13 @@ public static class Builder
 
     protected Long keyStoreReloadIntervalSecs;
 
+    public Http2SolrClient.Builder withListenerFactory(List<HttpListenerFactory> listenerFactory) {
```

Comment:
@epugh I know you worked on SolrClient immutability efforts.  Here, specific to Http2SolrClient, the "listenerFactory" list can now be set on the builder.  The setter should probably also be marked Deprecated.  Do you think this should be separated to another PR?